publishable/lang/pl/seeders.php

```diff
@@ -20,8 +20,8 @@
         'parent'           => 'Rodzic',
         'password'         => 'Hasło',
         'post_image'       => 'Grafika Postu',
-        'remember_token'   => 'Remember Token',
-        'role'             => 'Role',
+        'remember_token'   => 'Token Zapamiętaj',
```

Comment:
Nie mam lepszej propozycji. "Token Zapamiętaj" brzmi strasznie 'po polskiemu'. "Token Zapamiętaj mnie" albo "Przedłużony token sesji" lepiej oddają wartość tej kolumny, ale są trochę przydługie... z drugiej strony nikt o zdrowych zmysłach nie będzie wyświetlał tej kolumny na browse, więc może to nie przeszkadza?

Reply:
Tutaj właśnie miałem spory obawy, jak zrobić to sensownie. Myślałem też o Token "zapamiętaj mnie", ale nie miałem okazji sprawdzić, czy na pewno w całym kodzie są escape'owane cudzysłowy. Przedłużony token sesji brzmi bardzo fajnie :)